src/OrchardCore.Modules/OrchardCore.Media/Controllers/AdminController.cs

```diff
@@ -115,7 +122,10 @@ public async Task<ActionResult<IEnumerable<object>>> GetMediaItems(string path,
             var allowedExtensions = GetRequestedExtensions(extensions, false);
 
             var allowed = _mediaFileStore.GetDirectoryContentAsync(path)
-                .WhereAwait(async e => !e.IsDirectory && (allowedExtensions.Count == 0 || allowedExtensions.Contains(Path.GetExtension(e.Path))) && await _authorizationService.AuthorizeAsync(User, Permissions.ManageMediaFolder, (object)e.Path))
+                .WhereAwait(async e =>
+                    !e.IsDirectory &&
+                    (allowedExtensions.Count == 0 || allowedExtensions.Contains(Path.GetExtension(e.Path))) &&
```

Comment:
Personally will start the condition with the operator so it starts with &&. This was if you want to comment out a condition you can just comment out a single line

Reply:
Yeah, and the other such cases are like that in this class, so I'm changing it.